src/arrow_reader/column.rs

```diff
@@ -19,19 +20,24 @@ pub mod float;
 pub mod int;
 pub mod present;
 pub mod string;
+pub mod r#struct;
```

Comment:
Could we rename to not have to use raw identifiers? Maybe struct_column for example

Reply:
Yes, I will rename it.